test/Sentry.Azure.Functions.Worker.Tests/SentryFunctionsWorkerMiddlewareTests.cs

```diff
@@ -63,7 +63,7 @@ public async Task Original_exception_rethrown()
     }
 
     [Fact]
-    public async Task Transaction_name_and_operation_set()
+    public async Task Transaction_properties_set()
```

Comment:
```suggestion
    public async Task Transaction_PropertiesAreSet()
```
Since it's already getting renamed we could go with the [naming convention](https://github.com/getsentry/sentry-dotnet/blob/main/CONTRIBUTING.md#naming-tests).